master/setup.py

```diff
@@ -418,6 +418,9 @@ def define_plugin_entries(groups):
         'bundle': [
             'buildbot-www',
             'buildbot-slave'
+        ],
+        'docs': [
+            'sphinxcontrib-blockdiag>=0.8',
```

Comment:
this is not the same deps.
old version said 

sphinxcontrib-blockdiag (implicit latest)
'docutils>=0.8'  forces upgrade of docutils